src/styles/global.css

```diff
@@ -1,3 +1,7 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+body {
+    @apply bg-blue-gradient;
+}
```

Comment:
- should we apply the background to the top most div in `_app.tsx`?

Reply:
can't as we have to define it's CSS in tailwind file 
as bg-blue-gradient is used at two places

Reply:
removed add in global CSS file